[32mINFO    [0m sqlalchemy.engine.Engine:base.py:1848 SELECT bookmazon."user".user_account AS bookmazon_user_user_account 
FROM bookmazon."user" JOIN bookmazon.password_reset_tokens ON bookmazon."user".user_email = bookmazon.password_reset_tokens.user_email 
WHERE bookmazon.password_reset_tokens.token = %(token_1)s 
 LIMIT %(param_1)s
[32mINFO    [0m sqlalchemy.engine.Engine:base.py:1848 [generated in 0.00022s] {'token_1': '3GTPtl4Nwj3zZcqsDIt0hDIA8xcTmZio2q8ks8RMTKM', 'param_1': 1}
[32mINFO    [0m sqlalchemy.engine.Engine:base.py:2692 ROLLBACK
[31m[1mERROR   [0m C:\Users\user\PycharmProjects\pythonProject\bookmazon\utils\logger.py:UserService.py:472 Failed to change user password: (psycopg2.errors.InFailedSqlTransaction) current transaction is aborted, commands ignored until end of transaction block

[SQL: SELECT bookmazon."user".user_account AS bookmazon_user_user_account 
FROM bookmazon."user" JOIN bookmazon.password_reset_tokens ON bookmazon."user".user_email = bookmazon.password_reset_tokens.user_email 
WHERE bookmazon.password_reset_tokens.token = %(token_1)s 
 LIMIT %(param_1)s]
[parameters: {'token_1': '3GTPtl4Nwj3zZcqsDIt0hDIA8xcTmZio2q8ks8RMTKM', 'param_1': 1}]
(Background on this error at: https://sqlalche.me/e/20/2j85)